pkg/relevancymanager/v1/testdata/nginx-spdx-filtered.json

```diff
@@ -643,13 +644,9154 @@
               }
             ]
           }
+        },
+        {
+          "id": "749687e02ed2255c",
+          "name": "debconf",
+          "version": "1.5.82",
+          "type": "deb",
+          "foundBy": "dpkgdb-cataloger",
+          "locations": [
+            {
+              "path": "/usr/share/doc/debconf/copyright",
+              "layerID": "sha256:1c3daa06574284614db07a23682ab6d1c344f09f8093ee10e5de4152a51677a1",
+              "accessPath": "",
+              "annotations": {
+                "evidence": "supporting"
+              }
+            },
+            {
+              "path": "/var/lib/dpkg/info/debconf.conffiles",
+              "layerID": "sha256:1c3daa06574284614db07a23682ab6d1c344f09f8093ee10e5de4152a51677a1",
+              "accessPath": "",
+              "annotations": {
+                "evidence": "supporting"
+              }
+            },
+            {
+              "path": "/var/lib/dpkg/info/debconf.md5sums",
+              "layerID": "sha256:1c3daa06574284614db07a23682ab6d1c344f09f8093ee10e5de4152a51677a1",
+              "accessPath": "",
+              "annotations": {
+                "evidence": "supporting"
+              }
+            },
+            {
+              "path": "/var/lib/dpkg/status",
+              "layerID": "sha256:dcb816d1345ed4a429dec632c0933e2af8887b808259985bfc555d155d8c615d",
+              "accessPath": "",
+              "annotations": {
+                "evidence": "primary"
+              }
+            }
+          ],
+          "licenses": [
+            {
+              "value": "BSD-2-clause",
+              "spdxExpression": "BSD-2-Clause",
+              "type": "declared",
+              "urls": [],
+              "locations": [
+                {
+                  "path": "/usr/share/doc/debconf/copyright",
+                  "layerID": "sha256:1c3daa06574284614db07a23682ab6d1c344f09f8093ee10e5de4152a51677a1",
+                  "accessPath": ""
+                }
+              ]
+            }
+          ],
+          "language": "",
+          "cpes": [
+            "cpe:2.3:a:debconf:debconf:1.5.82:*:*:*:*:*:*:*"
+          ],
+          "purl": "pkg:deb/debian/debconf@1.5.82?arch=all\u0026distro=debian-12",
+          "metadata": {
+            "package": "debconf",
+            "source": "",
+            "version": "1.5.82",
+            "sourceVersion": "",
+            "architecture": "all",
+            "maintainer": "Debconf Developers \u003cdebconf-devel@lists.alioth.debian.org\u003e",
```

Comment:
@dwertent it seems we have unicode encoding issues

Reply:
That is probably the way I saved the file :(